### PR TITLE
Return `this` in `.start()` and `.update()`

### LINKED ISSUE #43: jsdoc contradiction to code in GameEntity
I just created typescript definitions for all types in yuka (I wonder if I should submit them as PR here) and found that we have a contradiction in return types in `GameEntity.js`
https://github.com/Mugen87/yuka/blob/6314fd6f0297197f05342361d36fd5825c3b171e/src/core/GameEntity.js#L202-L216

See, we define that we are returning `this`, while we are in fact a void method.

I wonder what the real intention is here, as I personally do not see any benefit in chaining either method (To be honest, most methods in GameEntity won't be chained in real life applications).

Shall we now add the missing `return this` or update the jsdoc to reflect `@return void`?

### PR DESCRIPTION
**Pull request by discordier**
Fixes #43

### CODE REVIEW

File: build/yuka.js [hunk 1]
```diff
@@ -3125,7 +3125,11 @@
 		*
 		* @return {GameEntity} A reference to this game entity.
 		*/
-		start() {}
+		start() {
```
Comment (Mugen87) [inline build/yuka.js:3128]: Do you mind to undo the changes to the build files?

Comment (discordier) [reply build/yuka.js]: done and force pushed.
